apps/datahub/src/app/search/search-results/search-results.component.html

```diff
@@ -8,17 +8,16 @@
     mel.datahub.search.hits.found
   </div>
   <mel-datahub-results-list-grid
-    [numberOfResults]="18"
+    [numberOfResults]="pageSize"
   ></mel-datahub-results-list-grid>
-  <div class="flex justify-center mt-8">
-    @if((searchFacade.isLoading$ | async) === false &&
-    (searchFacade.isEndOfResults$ | async) === false){
+  <div class="flex justify-center mt-16">
+    @if((searchFacade.isLoading$ | async) === false){
     <ng-container>
```

Comment:
I know the `<ng-container>` was there before, but I am questioning its existence? Do you know why we need it? I did a quick research and it says, that it is useful when 

> you want to conditionaly append a group of elements (ie using *ngIf="foo") in your application but don't want to wrap them with another element

(https://stackoverflow.com/a/40108573)

Reply:
I don't know why it was there TBH... I will remove it if it works without !